package/Dockerfile

```diff
@@ -19,34 +19,20 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM debian:stretch-slim
+FROM alpine:3.14
 
-ARG DEBIAN_FRONTEND=noninteractive
-
-COPY clean-apt /usr/bin
-COPY clean-install /usr/bin
 COPY Gemfile /Gemfile
 COPY Gemfile.lock /Gemfile.lock
 
-# 1. Install & configure dependencies.
-# 2. Install fluentd via ruby.
-# 3. Remove build dependencies.
-# 4. Cleanup leftover caches & files.
-# RUN BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev" \
-RUN BUILD_DEPS="make ruby-dev build-essential autoconf automake libtool libsnappy-dev" \
-    && clean-install $BUILD_DEPS \
-                     ca-certificates \
-                     libjemalloc1 \
-                     ruby \
+RUN apk update \
+    && apk add --no-cache ca-certificates ruby ruby-dev ruby-irb ruby-etc ruby-webrick build-base \
```

Comment:
are all of these required at runtime? could we put some of them in a virtual package and remove them at the end?

Reply:
Yes. Ruby has both compile time and runtime dependencies.